src/main/scala/piconot/internal/Empty.scala

```diff
@@ -0,0 +1,47 @@
+package piconot.internal
```

Comment:
I know that it become less clean than the ideal language design, but I still don't think that I dislike what it ended up as. It's very clear to me what is the initial information and how that informs what picobot is going to do next.